modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -27,6 +27,7 @@
 # globals:
 JFR_ENABLED = "jfr_enabled"
 IGNITE_TEST_CONTEXT_CLASS_KEY_NAME = "IgniteTestContext"
+SAFEPOINT_LOGS_DISABLED = "SafePointLogsDisabled"
```

Comment:
Please use the snake case for global variable name (in other words follow the existing approach of already existing variables: jfr_enabled, ignite_versions, jmx_remote etc.)

Reply:
Done